Description: Change log level from DEBUG to INFO
Author: Gernot Schulz <gernot@intevation.de>
Last-Update: 2016-07-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/intelmq/etc/system.conf
+++ b/intelmq/etc/system.conf
@@ -1,4 +1,4 @@
 {
-    "logging_level": "DEBUG",
+    "logging_level": "INFO",
     "logging_path": "/opt/intelmq/var/log/"
 }
